subsys/testsuite/Kconfig

```diff
@@ -144,4 +144,15 @@ config TEST_ARM_CORTEX_M
 	  the testing suite to utilize these exceptions, in tests.
 	  Note that by default, when building with ARM_SECURE_FIRMWARE
 	  set, these exceptions are set to target the Non-Secure state.
+
+config TEST_BUSY_SIM
+	bool "Enable busy simulator"
+	depends on TEST
+	select ENTROPY_GENERATOR
+	select RING_BUFFER
+	select COUNTER
+	help
+	  It simulates cpu load by using counter device to generate random
+	  interrupts with random busy looping in the interrupt.
```

Comment:
The term "random interrupts" is a bit ambiguous. I think something like "interrupts with random intervals" could make it more clear.